### DISCUSSION TITLE
Commercial use when running our own API

### DISCUSSION THREAD

**Discussion by gbstox** [open-meteo/open-meteo#222] [General]
Hello,

Awesome API!
A quick clarification question:  does the non-commercial use apply to both your API and if we run our own API[ using your code?](https://github.com/open-meteo/open-meteo/blob/main/docs/getting-started.md) Or does it only apply if we hit your API directly?

In other words, can we use this to run our own API and use it for commercial purposes?

Thank you!

**Comment by patrick-zippenfenig** [comment]
Hi,

yes you can run your own API directly and use is commercially. Code is licensed with AGLP therefore you have to publish all modifications you do to the source code. In the future there will also be commercial options at Open-Meteo. You can contact me via email to get more informations.

Best, Patrick
